mkdocs.yml

```diff
@@ -4,67 +4,70 @@ nav:
     - Home:
         - Introduction: index.md
         - Glossary: glossary.md
-        - Getting Started: setup_getting_started.md
         - License: license.md
-    - Overview:
+        - Friends: friends.md
         - F.A.Q.: faq.md
+    - User Guide:
+        - Getting Started:
+            - Downloading an image: setup_getting_started.md
+    - Development Guide:
         - Backend Services: backend.md
```

Comment:
this backend intro fits in both sections, perhaps it should be split at some point once more info is added that warrants 2 pages

Reply:
On it